src/relay/backend/contrib/tensorrt/common_utils.cc

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*!
+ * \file src/relay/backend/contrib/tensorrt/common_utils.cc
+ * \brief Utility functions used by compilation and runtime.
+ */
+
+#include "common_utils.h"
+
+namespace tvm {
+namespace relay {
+namespace contrib {
+
+std::vector<int> GetShape(const Type& type) {
```

Comment:
It is implemented here:

https://github.com/neo-ai/tvm/blob/5189c6be00c1b4871d6847e404614a0515cb212f/src/relay/backend/contrib/codegen_c/codegen_c.h#L174

Reply:
Hey @zhiics , that method is a class method of CodegenCBase. I don't see a good way to use that.

Reply:
hmm, never mind. There are a few other places using it. Maybe we can create a free function for it. We can ignore it for now.

Reply:
You can probably just implemented it in the .h file as a static inline function though. It looks to me that we don't need to have a separate cc file for this single function

Reply:
I tried implementing it inline, but since it is used by enable_tensorrt and tensorrt_builder it created a linker error.